packages/react-scheduler/src/__tests__/SchedulingUMDBundle-test.internal.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @jest-environment node
+ */
+'use strict';
+
+describe('Scheduling UMD bundle', () => {
+  beforeEach(() => {
+    // Fool SECRET_INTERNALS object into including UMD forwarding methods.
+    global.__UMD__ = true;
+
+    jest.resetModules();
+  });
+
+  function compareAPIS(apis) {
+    apis = apis.map(api => Object.keys(api).sort());
+    for (let i = 1; i < apis.length; i++) {
+      expect(apis[0]).toEqual(apis[i]);
```

Comment:
Might be a bit clearer if the "source of truth" API is passed as first argument. Then you can remove the `let i = 1` special case. Also errors would be better if you do

```js
expect(apis[i]).toEqual(nodeAPI);
```

because it's UMD/Secrets API that are *actual* and the original source code that is *expected*.